engines/scumm/gfx.h

```diff
@@ -69,10 +69,12 @@ struct CameraData {
 
 /** Virtual screen identifiers */
 enum VirtScreenNumber {
-	kMainVirtScreen = 0,	// The 'stage'
-	kTextVirtScreen = 1,	// In V0-V3 games: the area where text is printed
-	kVerbVirtScreen = 2,	// The verb area
-	kUnkVirtScreen = 3		// ?? Not sure what this one is good for...
+	kMainVirtScreen,	// The 'stage'
```

Comment:
Please return the enum values back, because they're used by the scripts.

Reply:
Ah, these (integer) values are actually the same as before, they're just not explicitly stated since it seemed redundant to do so, but I can restore the explicit value assignments no problem.